user_sync/app.py

```diff
@@ -105,11 +105,16 @@ def init_log(logging_config):
     }
     
     console_log_level = level_lookup.get(options['console_log_level'])
-    if (console_log_level != None):
-        console_log_handler.setLevel(console_log_level)
-    
+    if (console_log_level == None):
+        console_log_level = logging.DEBUG
```

Comment:
After a security review with our audit team, they requested that the default logging level be Info rather than debug.  Affects message a couple of lines down also, and file log level also.